docs/configuration/modules/remote-service.md

```diff
@@ -31,6 +31,28 @@ to connect to the remote server. The default value of this property is `true`.
 The `port` property is an optional property that defines the connection port. By default, if SSL/TLS protocol
 is used, the default value of this property is `443`. Otherwise, it is `80`.
 
+## autoreconnect-enabled
+
+The `autoreconnect-enabled` property is an optional property that defines if the server should be automatically reconnected when the connection is lost. The default value of this property is `false`.
+
+## reconnection-initial-interval
+
+The `reconnection-initial-interval` property is an optional property that defines the initial interval (in seconds) before the first reconnection. Its default values is 5.
+
+## reconnection-interval-mutiplier
+
+The `reconnection-interval-mutiplier` property is an optional property that is used to increase the waiting interval between two reconnection attempts, for example if we wait 5 seconds for the first retry, we will then wait 5*reconnection-interval-mutiplier seconds before the second retry. Its default values is 2.
+
+## reconnection-max-interval
+
+The `reconnection-max-interval` property is an optional property that that defines the max interval in seconds between two reconnections. Its default value is 3600.
+With the default parameters, there would be no reconnection attempt.
+With autoreconnect-enabled set to true and the other parameters set to default values, in the case of a disconnection, the reconnection attempts would occur after 5", then 10", 20", 40", 1'20", 2'40" .... etc incresing until a maximum of one hour interval. after reconnection max intenal, the reconnection will be re-attempted every hour.
+
+## reconnection-max
+
+The `reconnection-max` property is an optional property that that defines the maximum times in seconds before abandoning the attempts to reconnect. Its default value is Integer.MAX_VALUE.
```

Comment:
put back-ticks around the default value : *Its default value is `Integer.MAX_VALUE`.

Reply:
Ok It's done